docs/CI-CD.md

```diff
@@ -0,0 +1,93 @@
+# EVE CI/CD infrastructure
+
+EVE development happens on GitHub and we use GitHub's Actions as the core of our CI/CD infrastructure.
+Within GitHub actions we try to do as much as possible with the stock `ubuntu-20.04` build VM that
```

Comment:
I'd probably rewrite to say "stock Github ubuntu image (which as of [date] is ubuntu-20.04)" or something similar otherwise it will be out of date before you know it. Plus there are no guarantees that today's 20.04 was the same as yesterday's 20.04.

Reply:
done